README.md

```diff
@@ -30,6 +30,20 @@ Each sample demonstrates one feature of the SDK, together with tests.
 - [**Basic mTLS hello world**](./helloworldmtls): Simple example of a
   Workflow Definition and an Activity Definition using mTLS like Temporal Cloud.
 
+## Building sampes
```

Comment:
I don't think we want to promote platform-specific build steps. In fact, we should probably go the other direction and remove the Makefile altogether.

Reply:
> I don't care much as long as there is a solution. What solution do you suggest instead of Makefile? Mage?

Just use `go run`

> My reasoning is that I wanted to build/run only a single sample

See the README for whatever sample you're talking about which will tell you how to run the sample